platform/config/__tests__/ensureDeepObject.test.ts

```diff
@@ -4,7 +4,7 @@ test('flat object', () => {
   const obj = {
     'foo.a': 1,
     'foo.b': 2
-  }
+  };
```

Comment:
I just haven't put my IDE into rant-mode with TypeScript yet ;) We should definitely be consistent with adding them. Hopefully we'll add [Prettier](https://github.com/prettier/prettier) soon, which handles it for us